src/test/java/org/heigit/ohsome/ohsomeapi/controller/GetControllerTest.java

```diff
@@ -690,12 +714,12 @@ public void contributionsLatestCountTest() {
     TestRestTemplate restTemplate = new TestRestTemplate();
     ResponseEntity<JsonNode> responseAggregation =
         restTemplate.getForEntity(server + port
-            + "/contributions/latest/count?bboxes=8.67,49.39,8.71,49.42"
-            + "&filter=type:way and natural=*&format=json&time=2014-01-01/2017-01-01/P1Y",
-        JsonNode.class);
+                + "/contributions/latest/count?bboxes=8.67,49.39,8.71,49.42"
+                + "&filter=type:way and natural=*&format=json&time=2014-01-01/2017-01-01/P1Y",
+            JsonNode.class);
     ResponseEntity<JsonNode> responseExtraction = restTemplate.getForEntity(server + port
-        + "/contributions/latest/bbox?bboxes=8.67,49.39,8.71,49.42&filter=type:way and natural=*"
-        + "&properties=tags&time=2014-01-01,2017-01-01",
+            + "/contributions/latest/bbox?bboxes=8.67,49.39,8.71,49.42&"
+            + "filter=type:way and natural=*&properties=tags&time=2014-01-01,2017-01-01",
```

Comment:
the extra spacing seems superfluous here, or what's the reason for changing this?